src/structurizr/model/model.py

```diff
@@ -94,6 +96,7 @@ def __init__(
         people: Optional[Iterable[Person]] = (),
         software_systems: Iterable[SoftwareSystem] = (),
         deployment_nodes: Iterable[DeploymentNode] = (),
+        implied_relationship_strategy: Optional[Callable[[Relationship], None]] = None,
```

Comment:
Shouldn't the default here rather be `default_implied_relationship_strategy`? Would also allow you to remove that extra check for `None`.

Reply:
I think I had a circular import problem originally, but that's gone now so I'll do that, thanks.